src/Hazelcast.Net/CP/CPSessionManager.cs

```diff
@@ -240,7 +240,8 @@ public async ValueTask DisposeAsync()
 
             try
             {
-                await _heartbeating.CfAwaitCanceled();
+                if (_heartbeating != null)
```

Comment:
Is this related to this issue fix? Looks like another unrelated fix to me.

Reply:
I faced with an error on unisocket mode. I didn't dig it but I think because connection is not established, heartbeat is failing during disposing.

Reply:
I don't understand. We cannot add a null check without understanding why it's null. It may be hiding another issue.

Reply:
Okay. Understood the reason. `_heartbeating` is a `Task` to keep track of hear beating process. It is created when needed but disposed all the time. Fixed at 529b03865325286f2dc1300d878f28465c62fcf9